backend/development.ini.sample

```diff
@@ -1,267 +1,373 @@
+# email_reply feature
 email.notification.reply_to.email = reply+{content_id}@trac.im
+# tagging reference header of email with content_id is nice to have
+# all modification to a same content in a same email client thread
+# Note: reference doesn't need to be an existing email account but should
+# be a email like string.
 email.notification.references.email = thread+{content_id}@trac.im
-email.notification.content_update.template.html = %(here)s/tracim_backend/templates/mail/content_update_body_html.mak
-email.notification.content_update.template.text = %(here)s/tracim_backend/templates/mail/content_update_body_text.mak
-email.notification.created_account.template.html = %(here)s/tracim_backend/templates/mail/created_account_body_html.mak
-email.notification.created_account.template.text = %(here)s/tracim_backend/templates/mail/created_account_body_text.mak
-email.notification.reset_password_request.template.html = %(here)s/tracim_backend/templates/mail/reset_password_body_html.mak
-email.notification.reset_password_request.template.text = %(here)s/tracim_backend/templates/mail/reset_password_body_text.mak
-# Note: items between { and } are variable names. Do not remove / rename them
 email.notification.content_update.subject = [{website_title}] [{workspace_label}] {content_label} ({content_status_label})
 email.notification.created_account.subject = [{website_title}] Created account
-# processing_mode may be sync or async
-email.notification.processing_mode = sync
-email.notification.smtp.server = your_smtp_server
-email.notification.smtp.port = 25
-email.notification.smtp.user = your_smtp_user
-email.notification.smtp.password = your_smtp_password
-
-## Email sending configuration
-# processing_mode may be sync or async,
-# with async, please configure redis below
-email.processing_mode = sync
-# email.async.redis.host = localhost
-# email.async.redis.port = 6379
-# email.async.redis.db = 0
 
-# Email reply configuration
+### Templates ###
+email.notification.content_update.template.html = %(email.template_dir)s/content_update_body_html.mak
+email.notification.content_update.template.text = %(email.template_dir)s/content_update_body_text.mak
+email.notification.created_account.template.html = %(email.template_dir)s/created_account_body_html.mak
+email.notification.created_account.template.text = %(email.template_dir)s/created_account_body_text.mak
+email.notification.reset_password_request.template.html = %(email.template_dir)s/reset_password_body_html.mak
+email.notification.reset_password_request.template.text = %(email.template_dir)s/reset_password_body_text.mak
+
+####
+# EMAIL-REPLY
+####
+
+# Email reply configuration:
+# Reply by email feature: allow user to answer directly to tracim in reply
+# to notification.
 # This need api.key with non-empty value
+# This feature need additional daemons: see README for more info
+## available choice: True or False
 email.reply.activated = False
+
+### IMAP ###
```

Comment:
Add a comment to explain what is this IMAP configuration. Some user may have difficulties with architecture of mechanism.

Reply:
@buxx Do you think is good:

; With imap you save your email on server and you can access directly to email from anywhere.
; Whis this feature, tracim not download email localy.

Reply:
Suggestion:

> Tracim email reply feature will read all mails in following IMAP to be able to transform email responses into Tracim app comments.